Paging Data Source Example/Pods/Target Support Files/Pods-Paging Data Source Example/Pods-Paging Data Source Example-acknowledgements.plist

```diff
@@ -41,7 +41,7 @@ THE SOFTWARE.
 		</dict>
 		<dict>
 			<key>FooterText</key>
-			<string>https://tldrlegal.com/license/mit-license
+			<string>The MIT License (MIT)
```

Comment:
That really happened back at https://github.com/vokal/Vokoder/commit/eca1bb1860602b792c5b76bbfffedb844d77d80b.